File: Data Structures & Algorithms/minimum-size-subarray-sum/submission-1.java
class Solution {
    public int minSubArrayLen(int target, int[] nums) {
        int min = Integer.MAX_VALUE;
        int l = 0;
        int currSum = 0;
        for(int r = 0;r < nums.length;r++){
            currSum += nums[r];

            while(currSum >= target && l<=r){
                if(currSum >= target) min = Math.min(r-l+1,min);
                currSum -= nums[l];
                l++;
            }
        }

        return min == Integer.MAX_VALUE ? 0: min;
    }
}

File: Data Structures & Algorithms/merge-triplets-to-form-target/submission-1.py
class Solution:
    def mergeTriplets(self, triplets: List[List[int]], target: List[int]) -> bool:
        
        c1,c2,c3 = -1,-1,-1

        for a,b,c in triplets:
            if a <= target[0] and b <= target[1] and c <= target[2]:
                c1 = max(a,c1)
                c2 = max(b,c2)
                c3 = max(c,c3)
        return c1 == target[0] and c2 == target[1] and c3 == target[2]